source/mod_particles.f90

```diff
@@ -56,7 +55,7 @@ subroutine part_applyClosedOrbit
     xv2(1:napx)   = xv2(1:napx)   +   clo(2)*real(idz(2),fPrec)
     yv2(1:napx)   = yv2(1:napx)   +  clop(2)*real(idz(2),fPrec)
```

Comment:
Not sure about the units of the clo(2) is it x'?

Reply:
I think it is. The code here was moved from main_cr some time last year.